### ISSUE TITLE
Respect alignment

### ISSUE THREAD

**Issue by tfaehnle** [bkook/BKMoneyKit#6]
BKCurrencyTextField does not respect the horizontal text alignment set via storyboard

**Comment by bkook** [comment]
Is this issue resolved by removing text alignment code like below?
https://github.com/bkook/BKMoneyKit/pull/7

**Comment by tfaehnle** [comment]
Yes, it is. (In my pull request, only the second commit is the one that should be merged -- the first one removes a bunch of comments for some reason)

**Comment by fuer4869** [comment]
How can i match the chinese bankcard with BKCardNumberField?
 For example  6222 6001 1003 9982 714
